src/test/java/teammates/test/cases/ui/browsertests/StudentProfilePageUiTest.java

```diff
@@ -122,32 +122,32 @@ private void testActions() {
 
         ______TS("Typical case: no picture");
 
-        profilePage.editProfileThroughUi("", "short.name", "e@email.tmt", "inst", "Usual Nationality",
+        profilePage.editProfileThroughUi("", "short.name", "e@email.tmt", "inst", "Singaporean",
                                          "male", "this is enough!$%&*</>");
-        profilePage.ensureProfileContains("short.name", "e@email.tmt", "inst", "Usual Nationality",
+        profilePage.ensureProfileContains("short.name", "e@email.tmt", "inst", "Singaporean",
                                           "male", "this is enough!$%&*</>");
         profilePage.verifyStatus(Const.StatusMessages.STUDENT_PROFILE_EDITED);
-        
+
```

Comment:
It might be good to add a test case where the default option (the one with label "--- select ---") is selected, so we know that it is properly handled.